module/config.mjs

```diff
@@ -5316,11 +5316,26 @@ function addPower(powerDescription6e, powerOverrideFor5e) {
         {
             key: "TRANSFORM",
             type: ["attack", "standard"],
-            behaviors: ["attack"],
+            behaviors: ["attack", "dice"],
             perceivability: "obvious",
             duration: "instant",
             target: "target's DCV",
             range: HERO.RANGE_TYPES.STANDARD,
+            costPerLevel: function (item) {
+                switch (item.system.OPTIONID) {
+                    case "COSMETIC":
+                        return item.is5e ? 5 : 3;
+                    case "MINOR":
+                        return item.is5e ? 10 : 5;
+                    case "MAJOR":
+                        return item.is5e ? 15 : 10;
+                    case "SEVERE":
+                        return 15;
+                }
+                // This should never happen
+                console.error("Invalid TRANSFORM option ID: " + item.system.OPTIONID);
```

Comment:
You don't need to change it for this, but it is slightly nicer using the template literals. Consider for the future. String concat is definitely the old way to do things but you will see it throughout the code still.